atomic_reactor/plugins/pre_check_and_set_platforms.py

```diff
@@ -31,10 +37,32 @@ def __init__(self, tasker, workflow, koji_target):
 
         :param tasker: DockerTasker instance
         :param workflow: DockerBuildWorkflow instance
+        :param koji_target: str, Koji build target name
         """
         # call parent constructor
         super(CheckAndSetPlatformsPlugin, self).__init__(tasker, workflow)
         self.koji_target = koji_target
+        try:
+            self.goarch = get_platform_to_goarch_mapping(workflow)
+        except KeyError:
+            self.goarch = None
+
+    def validate_platforms(self, platforms):
+        """
+        Verify that a platform with GOARCH=amd64 is present. If it is not,
+        the tag will not be pullable by clients that do not have
+        support for the 'manifest list' type.
+        """
+        if self.goarch is None:
+            # Don't perform this check without a real reactor_config_map
```

Comment:
Maybe log that validation is being skipped?

Reply:
Added.